### ISSUE TITLE
Test browser.* namespace with Chrome & Edge

### ISSUE THREAD

**Issue by Jimadine** [Jimadine/chrome-ext-twitter2nitter#2]
Currently the extension uses the `chrome.*` namespace. The extension works in Firefox because:
> Firefox also supports the chrome.* namespace for APIs that are compatible with Chrome, primarily to assist with [porting](https://extensionworkshop.com/documentation/develop/porting-a-google-chrome-extension/) 

[Source](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Build_a_cross_browser_extension#:~:text=Firefox%20also%20supports%20the%20chrome.*%20namespace%20for%20APIs%20that%20are%20compatible%20with%20Chrome%2C%20primarily%20to%20assist%20with%20porting)

However, ideally the `browser.*` namespace should be used but it's unclear whether Chrome supports this:

> Note: The browser namespace is supported by Firefox and Safari. Chrome does not offer the browser namespace, until [Chrome bug 798169](https://crbug.com/798169) is resolved.

[Source](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Chrome_incompatibilities#:~:text=Note%3A%20The%20browser%20namespace%20is%20supported%20by%20Firefox%20and%20Safari.%20Chrome%20does%20not%20offer%20the%20browser%20namespace%2C%20until%20Chrome%20bug%20798169%20is%20resolved.)

[This github issue](https://github.com/w3c/webextensions/issues/113) suggests support for `browser.*` namespace was merged earlier in the year, so there's a fair chance the general release of Chrome now supports it.

Microsoft Edge appears to already support the `browser.*` namespace:

> Microsoft Edge uses the browser namespace, but doesn't yet support promise-based asynchronous APIs. In Edge, for the time being, asynchronous APIs must use callbacks.

[Source](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API#:~:text=Microsoft%20Edge%20uses%20the%20browser%20namespace%2C%20but%20doesn%27t%20yet%20support%20promise%2Dbased%20asynchronous%20APIs.%20In%20Edge%2C%20for%20the%20time%20being%2C%20asynchronous%20APIs%20must%20use%20callbacks.)

**Comment by Jimadine** [comment]
Seemingly Chromium 131.0.6778.86 doesn't have the needed change:
```
Uncaught ReferenceError: browser is not defined
```

**Comment by Jimadine** [comment]
Next best thing is:
```
const xbrowser = chrome ?? browser;
```
...as suggested [here](https://www.tadeasjun.com/blog/crossbrowser-extensions/).

**Comment by Jimadine** [comment]
The above choose `chrome` as the default namespace, whuch we don't want. If I change the order (`const xbrowser = browser ??chrome;`) it doesn't work in Chromium:
```
Uncaught ReferenceError: browser is not defined
```
This works:
```
if (typeof browser === "undefined") {
  var xbrowser = chrome;
} else {
  var xbrowser = browser;
}
```

**Comment by Jimadine** [comment]
Fixed in 3b38d4ac87c607614d054f767cb618b57696bb6c.
